src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -1028,6 +1043,13 @@ static void VehicleSpritePaintRestraints(
     vehicle_sprite_paint(session, vehicle, spriteNum, VehicleBoundboxes[carEntry->draw_order][boundingBoxNum], z, carEntry);
 }
 
+// Returns the opposite of the bank angle for reversed cars, normal bank angle otherwise
+static uint8_t GetBankRotationForDrawing(const Vehicle* vehicle)
```

Comment:
Generally this is seen as a paint function. The drawing functions are the ones that operate on the output of the paint graph. Just call it `GetPaintBankRotation`. I eventually want to rename bank rotation to roll as it makes things easier to understand (and is much shorter than bank rotation) amusingly that would then end up making this a GetPaintRoll 
![image](https://user-images.githubusercontent.com/1277401/217384469-61186c87-c2f8-4d6e-a282-506af574eaea.png)